custom_visibility_7_10/src/custom/data/visibility/FilterOpportunities.php

```diff
@@ -65,20 +65,30 @@ public function __construct(SugarBean $bean, $params = null)
     }
 
     /**
-     * {@inheritdoc}
+     * orvewrite parent method, SugarVisibility::addVisibilityWhere
+     *
+     * add visibility, i.e. condition, clauses to the WHERE part of the query
+     *
+     * this demo function adds new condition to where clause with sales_stage is NOT IN the list of $this->filterSalesStages
```

Comment:
I think "with" should be "when"

Reply:
It looks like you copied a lot of the comments from custom_visibility into this one.  Can you apply the suggestions I had above to this code as well?